content/Watsonx/WatsonxData/104.md

```diff
@@ -48,7 +49,8 @@ You will need MinIO credentials to log into and use the MinIO console. This incl
 7. Go back to the MinIO console. Click the row for the **iceberg-bucket** bucket.
 
   There are two folders listed, one for each schema that has been created in the catalog associated with this bucket (**iceberg_data**). Watsonx.data uses the schema name for the folder when you create it (such as when you created the schema **my_schema** earlier in the lab). Recall that when you created the schema **newschema** using the Presto CLI, you specified the location as 's3a://iceberg-bucket/newschema', which matches what you see here.
-
+  <QuizAlert text="Quiz material: pay attention to S3 objects structure"/>
```

Comment:
"S3 objects structure" -> "the S3 object's structure"

Reply:
done